app/views/shared/context_tables/_commodity.html.erb

```diff
@@ -17,6 +17,7 @@
     </dd>
     <dd class="govuk-summary-list__actions"></dd>
   </div>
+  <%= commodity_validity_dates(declarable) %>
```

Comment:
Not saying your approach is wrong, but shouldn't we just be consistent with the other rows?

Reply:
Maybe its worth refactoring them to be consistent with the helper approach? And maybe we should add a new helper for the goods nomenclature context summary lists?